modules/oe_content_person/config/install/field.field.node.oe_person.oe_person_documents.yml

```diff
@@ -0,0 +1,35 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_person_documents
+    - node.type.oe_person
+    - oe_content_sub_entity_document_reference.oe_document_reference_type.oe_document
+    - oe_content_sub_entity_document_reference.oe_document_reference_type.oe_publication
+  module:
+    - composite_reference
+    - entity_reference_revisions
+third_party_settings:
+  composite_reference:
+    composite: true
+id: node.oe_person.oe_person_documents
+field_name: oe_person_documents
+entity_type: node
+bundle: oe_person
+label: 'Articles and publications'
+description: 'Please list a set of documents and/or publications related to this person, by referencing already existing documents/publications.'
```

Comment:
@ademarco, in the spec https://webgate.ec.europa.eu/fpfis/wikis/display/EWPP/Person+Content+Type+Specification+-+Version+1.0.0 description has to be "Please list a set of documents and/or publications related to this person, by referencing already existing documents/publications. Alternatively, new document assets can be created directly from here.".

Currently we don't give ability to users create Media Document inside node. So this description is irrelevant and spec has to be updated or we need to use IEF complex widget for Document Document reference entity. In this case this widget will be on Consultation CT too.

<img width="1066" alt="Screenshot 2021-02-22 at 18 52 36" src="https://user-images.githubusercontent.com/1183951/108733132-42db7480-753f-11eb-866c-dc5992b5b8d7.png">